packages/minimongo/minimongo.js

```diff
@@ -363,9 +363,27 @@ _.extend(LocalCollection.Cursor.prototype, {
         if (self.collection.paused)
           return;
 
-        self.collection._observeQueue.queueTask(function () {
-          f.apply(context, args);
-        });
+        // Wrap a function as to not seem as if it's running as part
+        // of a method stub. One important consequence is that you'll
+        // be able to call `Meteor.call` within these wrapped
+        // functions and actually end up calling a server method (as
+        // opposed to just a client stub)
```

Comment:
I think there should be a pointer to #2315 here, noting that this is only a problem because we call observe callbacks synchronously, and a note in #2315 saying that the fix here (but not the test) can be reverted if we fix #2315.